Ginger/Ginger/ApplicationModelsLib/POMModels/PomElementsPage.xaml.cs

```diff
@@ -234,6 +234,10 @@ private void RemoveElementsToMappedBtnClicked(object sender, RoutedEventArgs e)
         internal void SetAgent(Agent agent)
         {
             mAgent = agent;
+            foreach(ElementInfo elemInfo in mElements)
+            {
+                elemInfo.ElementObject = null;
+            }
         }
```

Comment:
remove the object check IF in SeleniumDriver HighlightElement() to make sure we covering also scenario in which user changes only the page